docs/02-eksempler-bruksområder.adoc

```diff
@@ -2,13 +2,13 @@
 
 Noen eksempler på når spesifikasjonen skal brukes (ikke-uttømmende):
 
-* Beskrivelse av kvantifiserbar kvalitet til et datasett: Se eksemplet under egenskapen <<Datasett: har kvantifiserbart måleresultat (dqv:hasQualityMeasurement)>>.
+* Beskrivelse av kvantifiserbar kvalitet: Se eksemplet under egenskapen <<Datasett: har kvantifiserbart måleresultat (dqv:hasQualityMeasurement)>>.
 ** Når man refererer til en predefinert instans av et kvalitetsmål (f.eks. `dqvno:missingObjects` som er en predefinert instans av `dqv:Metric`), trenger man ikke å ta med kvalitets(del)dimensjonen som kvalitetsmålet er i (ettersom dette er predefinert sammen med det aktuelle kvalitetsmålet).
 
-* Beskrivelse av kvalitet i form av fritekst, inkl. brukertilbakemelding på kvalitet i fritekst: Se eksempel under egenskapen <<Datasett: har kvalitetsnote>>.
+* Beskrivelse av kvalitet i form av fritekst, inkl. brukertilbakemelding på kvalitet i fritekst: Se eksempel under egenskapen <<Datasett: har kvalitetsnote (dqv:hasQualityAnnotation)>>.
 
-* Beskrivelse av at kvaliteten til et datasett er i samsvar med gitt standard/spesifikasjon: Se eksempel under egenskapen <<Datasett: er i samsvar med>>.
+* Beskrivelse av at kvaliteten er i samsvar med gitt standard/spesifikasjon: Se eksempel under egenskapen <<Datasett: er i samsvar med (dct:conformsTo)>>.
 
-* Beskrivelse av at et datasett er autoritativt: Se eksempel under klassen <<Kvalitets Sertifikat (dqv:QualityCertificate)>>.
+* Beskrivelse av at et datasett er autoritativt: Se eksempel under klassen <<Kvalitetssertifikat (dqv:QualityCertificate)>>.
 
-Se ellers også Eksempel (ikke-normativt) bakerst i denne spesifikasjonen, som viser de ulike måtene å beskrive kvalitet på som denne spesifikasjonen dekker.
+Se ellers også <<eksemplet, eksemplet>> bakerst i denne spesifikasjonen, som viser de ulike måtene å beskrive kvalitet på som denne spesifikasjonen dekker.
```

Comment:
Eksempel to ganger? Mangler en 'e' i "eksemplet".

Reply:
to ganger fordi den første "ankeren", og den andre teksten som skal vises. "Eksemplet" er egentlig lovlig skriveform, men jeg retter uansett her.